cli/src/main/scala/scalaxb/compiler/xsd/GenSource.scala

```diff
@@ -616,22 +616,22 @@ class GenSource(val schema: SchemaDecl,
     val baseSym : Option[XsTypeSymbol] = decl.content match {case SimpTypRestrictionDecl(base, _) => Some(base) case _ => None}
     val baseType: Option[String      ] = baseSym.map(buildTypeName(_))
 
-    def makeEnum(enum: EnumerationDecl[_]) =
-      "case object " + buildTypeName(localName, enum, true) + " extends " + localName + 
-      " { override def toString = " + quote(enum.value.toString) + " }"
+    def makeEnum(`enum`: EnumerationDecl[_]) =
```

Comment:
Since I don't really care about compatibility in the generator code, we can call these `enumDecl` or something like that instead.